scipy/stats/tests/test_stats.py

```diff
@@ -4178,6 +4178,15 @@ def test_nan_policy(self):
         assert_almost_equal(stats.kruskal(x, x, nan_policy='omit'), (0.0, 1.0))
         assert_raises(ValueError, stats.kruskal, x, x, nan_policy='raise')
         assert_raises(ValueError, stats.kruskal, x, x, nan_policy='foobar')
+        
+    def test_large_no_samples(self):
+        # Test to see if large samples are handled correctly.
+        n = 50000
+        x = np.random.randn(n)
```

Comment:
before generating random numbers, you want to seed the random generator (with ``np.random.seed``). That way the test is fully reproducible.